scripts/common.py

```diff
@@ -712,3 +715,91 @@ def __enter__(self):
 
     def __exit__(self, etype, value, traceback):
         os.chdir(self.saved_path)
+
+
+@contextlib.contextmanager
+def managed_process(args, shell=False, **kwargs):
+    """Context manager for starting and stopping a process gracefully.
+
+    Args:
+        args: list(*). A sequence of program arguments, where the program to
+            execute is the first item. All items are converted to strings first.
+        shell: bool. Whether the command should be run inside of its own shell.
+        **kwargs: dict(str: *). Same kwargs as `subprocess.Popen`.
+
+    Yields:
+        psutil.Process. The process managed by the context manager.
+    """
+    if PSUTIL_DIR not in sys.path:
+        sys.path.insert(1, PSUTIL_DIR)
+    import psutil
+
+    str_args = (python_utils.UNICODE(arg).strip() for arg in args)
+    truthy_args = (arg for arg in str_args if arg)
```

Comment:
This is to remove empty strings right? I would maybe add a comment for explaining this. Or rewrite it to something like 
```suggestion
    # Transform args to strings and remove empty ones.
    str_args = (
        python_utils.UNICODE(arg).strip() for arg in args
        if python_utils.UNICODE(arg).strip()
    )
```

Reply:
Hmm I think the variable names should do that job, so I've tried updating them to be more clear.

How's that?

Reply:
Yes, better.